src/main/java/cz/centrum/haffner/SimpleTrainingDavid/AppServices/OneDayFileJsonParser.java

```diff
@@ -0,0 +1,122 @@
+package cz.centrum.haffner.SimpleTrainingDavid.AppServices;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import cz.centrum.haffner.SimpleTrainingDavid.DataTemplates.MetricsInfoDataSet;
+import org.springframework.stereotype.Component;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+public class OneDayFileJsonParser {
+    // TODO: net approach doesn´t work
+    // private static final String FILE_PATH = "https://github.com/DavidHaffner/SimpleTrainingDavid/tree/dev/logs/MCP_20180131.json";
+    private static final String FILE_PATH_PREFIX = "C://work/SimpleTrainingDavid/src/main/resources/MCPData/";
```

Comment:
Tahle konstanta bude sice nakonec v MetricsInfoService, ale i tak ji prosím inicializuj takhle:
private static final String FILE_PATH_PREFIX = OneDayFileJsonParser.class.getClassLoader().getResource("MCPData/").getPath();

(Zatím načítání soborů z netu řešit nebudeme, takže ve chvíli, kdy si to někdo stáhne k sobě a bude mít jinou adresářovou strukturu, tak mu to **resources** složku najde relativně podle cest v classloaderu.

Reply:
done